src/Symfony/Component/Console/Output/ConsoleOutputInterface.php

```diff
@@ -27,4 +29,11 @@
     public function getErrorOutput();
 
     public function setErrorOutput(OutputInterface $error);
+
+    /*
+     * Creates a new output section
+     *
+     * @return ConsoleSectionOutput
+     */
+    // public function section(); // TODO Add this method to the interface in Symfony 5
```

Comment:
I would not add this `section()` method in this interface as this is not really a required feature. If needed, we could create an additional interface, not sure if it's worth it though.